src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -185,4 +185,20 @@
       <VSDesignTimeBuild Condition="'$(BuildingInsideVisualStudio)'=='true' and '$(BuildingOutOfProcess)'=='false'">true</VSDesignTimeBuild>
     </PropertyGroup>
   </Target>
+  
```

Comment:
No need to mention FxTransform as that is legacy at this point. I would simply add the following comment in both places.

Providing a definition for __BlockReflectionAttribute in an assembly is a signal to the .NET Native toolchain to remove the metadata for all non-public APIs. This both reduces size and disables private reflection on those APIs in libraries that include this. The attribute can also be applied to individual public APIs to similarly block them.